lib/clients/aws-client/abstract-client.js

```diff
@@ -0,0 +1,21 @@
+const aws = require('aws-sdk');
+
+const CONSTANTS = require('@src/utils/constants');
+const stringUtils = require('@src/utils/string-utils');
+
+
+module.exports = class AbstractClient {
```

Comment:
should better be AbstractAwsClient. This is going to be our client to resolve credentials loading logic